build/_build/PrepareRelease/SetAllVersions.cs

```diff
@@ -7,14 +7,17 @@
 using System.IO;
 using System.Text;
 using System.Text.RegularExpressions;
+using Nuke.Common.IO;
 
 namespace PrepareRelease
 {
     public class SetAllVersions
     {
-        public SetAllVersions(string solutionDirectory)
+        public SetAllVersions(string solutionDirectory, string tracerVersion, bool isPrerelease)
```

Comment:
Nice. I will be able to clean up some code in my branch. I had to do something similar to pass versions from the command line:
https://github.com/DataDog/dd-trace-dotnet/blob/ab559b6be4742360d9265a4329a2505270842abb/build/tools/PrepareRelease/SetAllVersions.cs#L15-L20